src/main/java/ch/njol/skript/expressions/ExprSecCreateWorldBorder.java

```diff
@@ -22,16 +22,28 @@
 import java.util.List;
 import java.util.concurrent.atomic.AtomicBoolean;
 
-@Name("Create WorldBorder")
-@Description("Create a virtual worldborder. You can make this a real border by setting a world's worldborder to a virtual border.")
+@Name("Create Virtual WorldBorder")
+@Description({
+    "Creates a new virtual worldborder that can be customized inside a section.",
+    "Each time this expression is used, a fresh worldborder is created.",
+    "You can apply the customized worldborder to a player or to a world.",
+    "Setting a world's worldborder to a virtual border makes it a real, visible border."
```

Comment:
i think trying to make all these distinctions between real, fake, and virtual just makes it more confusing.
I'd rather see something like
```
Creates a new, unused world border. World borders can be assigned to either worlds or specific players.
Borders assigned to worlds apply to all players in that world.
Borders assigned to players apply only to those players, and different players can have different borders.
```